protos/batch.proto

```diff
@@ -17,6 +17,7 @@ syntax = "proto3";
 
 option java_multiple_files = true;
 option java_package = "sawtooth.sdk.protobuf";
+option go_package = "sawtooth_sdk/protobuf";
```

Comment:
I tried this first, but was getting a lot of complaints from Go's protoc plugin. Will try again